File: youpinhui-shop-web/src/main/webapp/js/service/sellerService.js
//Sevice
app.service('sellerService',function($http){
	    	
	 //test case , get all the data return a Json
	this.findAll=function(){
		return $http.get('../seller/findAll.do');		
	}
	//pagination
	this.findPage=function(page,rows){
		return $http.get('../seller/findPage.do?page='+page+'&rows='+rows);
	}
	//find one object
	this.findOne=function(id){
		return $http.get('../seller/findOne.do?id='+id);
	}
	//add 
	this.add=function(entity){
		return  $http.post('../seller/add.do',entity );
	}
	//update 
	this.update=function(entity){
		return  $http.post('../seller/update.do',entity );
	}
	//batch delete
	this.dele=function(ids){
		return $http.get('../seller/delete.do?ids='+ids);
	}
	//search
	this.search=function(page,rows,searchEntity){
		return $http.post('../seller/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: youpinhui-sellergoods-interface/src/main/java/com/youpinhui/sellergoods/service/SellerService.java
package com.youpinhui.sellergoods.service;
import java.util.List;

import com.youpinhui.entity.PageResult;
import com.youpinhui.entity.Result;
import com.youpinhui.pojo.TbSeller;


/**
 * provider interface
 * @author Administrator
 *
 */
public interface SellerService {

	/**
	 * find all
	 * @return
	 */
	public List<TbSeller> findAll();
	
	
	/**
	 * query according pagination 
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * Add
	*/
	public void add(TbSeller seller);
	
	
	/**
	 * update
	 */
	public void update(TbSeller seller);
	

	/**
	 * get one by id
	 * @param id
	 * @return
	 */
	public TbSeller findOne(String id);
	
	
	/**
	 * batch delete
	 * @param ids
	 */
	public void delete(String [] ids);

	/**
	 * fuzzy search 
	 * if the object  is not null , get the parameter from it as the search condition.
	 * @param pageNum current page number
	 * @param pageSize size of the page to display
	 * @return
	 */
	public PageResult findPage(TbSeller seller, int pageNum,int pageSize);
	
	/**
	 * update Seller status 
	 * 
	 * @param sellerId selected seller's id
	 * @param status  status to change
	 * 	
	 */
	public void updateStatus( String sellerId,String status);
	
	
	
}
